src/nbl/asset/interchange/CPLYMeshWriter.cpp

```diff
@@ -85,37 +85,84 @@ bool CPLYMeshWriter::writeAsset(io::IWriteFile* _file, const SAssetWriteParams&
 
```

Comment:
either you should just save the first meshbuffer and ignore the rest, or if you're gonna keep on quitting if meshbuffer count >1 then at least log an error for the user about it.

Reply:
corrected in following commit